src/app/Components/Buttons/features.tsx

```
@@ -3,26 +3,26 @@
 {/* Features Dropdown*/}
 {/* UI Component -- Dropdown menu displaying web features */}
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.

(lint/complexity/noUselessLoneBlockStatements)